### PR TITLE
add a description of the anatomical dataset

### PR DESCRIPTION
**Pull request by bpinsard**

### CODE REVIEW

File: source/DATASETS.md [hunk 1]
```diff
@@ -10,6 +12,16 @@ Note that BIDS session names have no meaning apart from being data acquired in t
 ## Participants
 Six healthy participants (aged 31 to 47 at the time of recruitment in 2018), 3 women (`sub-03`, `sub-04` and `sub-06`) and 3 men (`sub-01`, `sub-02` and `sub-05`) consented to participate in the Courtois Neuromod Project for at least 5 years. Three of the participants reported being native francophone speakers (`sub-01`, `sub-02` and `sub-04`), one as being a native anglophone (`sub-06`) and two as bilingual native speakers (`sub-03` and `sub-05`). All participants reported being in good general health, all had normal hearing and vision for their age, and all were MRI and MEG compatible.
 
+## anat
+
+The anatomical dataset intends to acquire longitudinal anatomical image of the brain and upper spinal cord at an approximate rate of 4 sessions a year.
```
Comment (pbellec) [inline source/DATASETS.md:17]:
```suggestion
The anatomical dataset includes longitudinal anatomical images of the brain and upper spinal cord at an approximate rate of 4 sessions a year. The primary intended use of this dataset is to monitor the structural stability of the brain of participants for the duration of the study. Many quantitative measures of brain structure can also be derived and included in analyses, such as gray matter morphometry, tractography or measures of myelination. 
```

File: source/DATASETS.md [hunk 2]
```diff
@@ -10,6 +12,16 @@ Note that BIDS session names have no meaning apart from being data acquired in t
...
+
+The MRI sequences are described in more detailed in :ref:`Brain anatomical sequences` :ref:`Spinal cord anatomical sequences`, including pdfs of the Siemens exam cards.
```
Comment (pbellec) [inline source/DATASETS.md:19]: Maybe include a high level summary of all sequences here (I am thinking of a one-liner).

File: source/DATASETS.md [hunk 3]
```diff
@@ -10,6 +12,16 @@ Note that BIDS session names have no meaning apart from being data acquired in t
...
+
+Brain T1w, T2w and DWI were copied from the HCP aging and development protocol for Prisma MRI scanner.
```
Comment (pbellec) [inline source/DATASETS.md:21]: Maybe add something like "Other sequences were selected and optimized by the Courtois NeuroMod team."

File: source/DATASETS.md [hunk 4]
```diff
@@ -10,6 +12,16 @@ Note that BIDS session names have no meaning apart from being data acquired in t
...
+
+All images covering the face were anonymized by zeroing the data in the face, teeth and ears regions with a custom mask warped from the MNI space based on a linear registration of the T1w brain MRI series.
```
Comment (pbellec) [inline source/DATASETS.md:23]: maybe link the tool / script which was used.

**Review by pbellec — CHANGES_REQUESTED** [review]

**Review by pbellec — APPROVED** [review]

**Review by julieaboyle1 — APPROVED** [review]

**Pull request merged**